tests/testthat/test_manifold.R

```diff
@@ -1 +1,19 @@
 # Unit tests for Manifold.
```

Comment:
Same as above: `context("Unit tests for Manifold")` as per testthat conventions:
https://vita.had.co.nz/papers/testthat.pdf

Testthats "contexts" seem to be made exactly for this.

Reply:
same @lundjack